remix-gql-tailwind/app/components/UserProfile/OrgList.tsx

```diff
@@ -13,7 +13,7 @@ function OrgList({ organizations }: OrgListProps) {
         {organizations.map(({ avatarUrl, login }) => (
           <div key={login} className={styles.listItem}>
             <img
-              src={avatarUrl}
+              src={`//${avatarUrl}`}
```

Comment:
This change is unnecessary—the `avatarUrl` returned from the GitHub API always has the protocol appended.